packages/beeq/src/components/tab-group/bq-tab-group.tsx

```diff
@@ -36,6 +36,12 @@ export class BqTabGroup {
   /** The size of the tab */
   @Prop({ reflect: true }) size: TTabSize = 'medium';
 
+  /** The direction that tab should be render */
+  @Prop({ reflect: true }) orientation?: TTabOrientation = 'horizontal';
+
+  /** The position that tab should be render */
+  @Prop({ reflect: true }) position?: TTabPosition = 'start';
```

Comment:
Could we use `placement` to maintain consistency with the Drawer, Dropdown, and Tooltip that use the same property name?